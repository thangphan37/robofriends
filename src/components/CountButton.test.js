import React from 'react';
import {shallow} from 'enzyme';
import renderer from 'react-test-renderer';
import CounterButton from './CountButton';

it('snapshot CountButton component', () => {
  const wrapper = renderer.create(<CounterButton />).toJSON();

  expect(wrapper).toMatchSnapshot();
});

it('should click count button', () => {
  const wrapper = shallow(<CounterButton />);
  wrapper.find('button').prop('onClick')();
  wrapper.find('button').prop('onClick')();

  expect(wrapper.find('button').text()).toEqual("2");
})