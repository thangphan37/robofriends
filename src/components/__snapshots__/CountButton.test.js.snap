// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot CountButton component 1`] = `
<div
  className="counter-container"
>
  <button
    onClick={[Function]}
  >
    0
  </button>
</div>
`;
